src/js/ble_hci_socket_gap.js

```diff
@@ -41,7 +41,7 @@ var util = require('util');
 
 var Hci = require('ble_hci_socket_hci');
 
-var isLinux = (process.platform === 'linux');
+// TODO: var isLinux = (process.platform === 'linux');
```

Comment:
isLinux is unused - I could delete it, but below are commented out lines that use it. Unfortunatelly whole BLE looks like WIP and in my personal opinion should not be commited. I want to avoid doing harm removing something useful for its author, that is why I just commented it out, with TODO marker.